support/validation.go

```diff
@@ -26,7 +27,8 @@ func ValidateFormatUUIDv4(s string) error {
 }
 
 // ValidateFormatDecimal checks it look like it's in a decimal shape
-func ValidateFormatDecimal(s string) error {
+func ValidateFormatDecimal(d decimal.Decimal) error {
```

Comment:
This is weird. There should be runtime marshaling errors that happen if the decimal format is incorrect if you're using decimal.Decimal or `decimaltype=shopspring` for this generator. Probably if the generator is configured in a way where a decimal type is being specified this regex approach still makes sense.